indico/modules/events/abstracts/templates/_abstract.html

```diff
@@ -303,3 +324,35 @@ <h3 class="abstract-title">
         </div>
     </div>
 {% endmacro %}
+
+{% macro render_abstract_comments(abstract) %}
+    {% if abstract.comments %}
+        <div id="abstract-comments" class="i-timeline">
+            {% for comment in abstract.comments %}
+                <div class="i-timeline-item">
+                    <div class="i-timeline-item-label action"></div>
+                    <div class="i-timeline-item-box header-indicator">
+                        <div class="i-box-header flex">
+                            <div class="stretch">
+                                {% trans name=comment.user.name, date=comment.created_dt|format_human_date -%}
+                                    <strong>{{ name }}</strong> commented on {{ date }}
+                                {%- endtrans %}
+                                {% if comment.modified_dt %}
+                                    {{ render_edited_hint(comment.modified_dt, comment.modified_by) }}
+                                {% endif %}
+                                {% if comment.can_edit(user) %}
```

Comment:
yuck! why not close the `div` here and then start the new one in the if can_edit block?